### PR TITLE
Newhub - responsivity, crossbrowser and cutoff fixes

### PR DESCRIPTION
**Pull request by anihilnine**
Responsivity improvements
Font cutoff fix
Tested on chrome, firefox, edge, client, mobile
Html and sass is rewritten to use more features of pug and sass
Entire cell is now clickable
Custom style for nomads
Support for cells with no links

### CODE REVIEW

File: public/styles/site/newshub.sass [hunk 1]
```diff
@@ -32,31 +39,35 @@ $headerColour: #6990e0
...
   width: 299px
   height: 602px
-  float: left
+  
 
 //columns
 .column1 
   max-width: 744px
-  float: left
+@media screen and (max-width: 760px)
```
Comment (norraxx) [inline public/styles/site/newshub.sass:157]: Use `{` for blocks `}`

File: templates/views/newshub.pug [hunk 2]
```diff
@@ -3,75 +3,76 @@ extends ../layouts/default
...
 
-      // normalArticles 
-      .column1
-      
-        for a in content.normalArticles
+        // sidebar right
+        if content.sidebarRight
+          #sidebar_right(style="background:#" + content.sidebarRight.backgroundcolor)
+            a(href=content.sidebarRight.linkUrl)
+              img(src=content.sidebarRight.imageUrl alt="")
```
Comment (norraxx) [inline templates/views/newshub.pug]: Do you need that alt? If yes, fill it, if not, don't use it

File: public/styles/site/newshub.sass (hunk 1 above)
Comment (anihilnine) [reply public/styles/site/newshub.sass:157]: this is scheduled for next release (complete reimplementation using proper sass syntax. html will change too)

File: templates/views/newshub.pug (hunk 2 above)
Comment (anihilnine) [reply templates/views/newshub.pug]: will remove in next release, lets just push this out

File: public/styles/site/newshub.sass [hunk 3]
```diff
@@ -3,13 +3,20 @@ $headerColour: #6990e0
...
   
 .newshub
   max-width: 1270px
   margin: auto
   padding-top: 25px
   color: rgb(214, 214, 214)
-
+  position: relative
+  
+.newshub_flex
```
Comment (norraxx) [inline public/styles/site/newshub.sass]: Do we mix block and flex displaying?

File: templates/views/newshub.pug [hunk 4]
```diff
@@ -3,75 +3,76 @@ extends ../layouts/default
...
+        // normalArticles 
+        .column1
+        
+          for a in content.normalArticles
 
-          else
-            // normal cell
-            .article_common
-              .article_common_content(class=a.customStyle)
+            if (a.position=="hero")
```
Comment (norraxx) [inline templates/views/newshub.pug]: IFs without brackets makes me nervous every time, when i'm not in python.

File: public/styles/site/newshub.sass (hunk 3 above)
Comment (ChaosWarrior80) [reply public/styles/site/newshub.sass]: It will be changed in next version. Probably today.

Comment (anihilnine) [reply public/styles/site/newshub.sass]: norraxx, this PR is just to fix the bugs that are in prod at the moment

the next release is a rewrite that offers more functionality but also addresses lots of implementation issues like not using sass nesting

can we please just get this approved and pushed out?

**Review by norraxx — APPROVED** [review]

**Comment by anihilnine** [comment]
found an issue that i need to fix before this goes out - edit: fixed

**Pull request merged**